peribolos/knative-extensions.yaml

```
@@ -1,7 +1,7 @@
 orgs:
   knative-extensions:
-    name: Knative Sandbox
-    description: Sandbox for projects that are part of the broader knative project
+    name: Knative Extensions
+    description: Extensions and plugins that extend Knative Serving and Eventing
```

nit: we've also got some client plugins, so maybe we want to be more generic?

```suggestion
    description: Extensions and plugins that extend the Knative project.
```

---

done